Emulate Media Queries

mdineen opened the pull request:
allow the emulator to apply style rules for -webkit-device-pixel-ratio media queries

Commit:
Emulate Media Queries

allow the emulator to apply style rules for -webkit-device-pixel-ratio media queries

gtanner commented:
Are you at the hub monday?

I wouldn't mind pairing up to clean some stuff up
Sent on the TELUS Mobility network with BlackBerry

mdineen commented:
You bet. See you there. 
Sent from my BlackBerry
+1 226 924 5232

-----Original Message-----
From: Gord Tanner notifications@github.com
Date: Fri, 7 Sep 2012 14:13:53 
To: blackberry/Ripple-UIRipple-UI@noreply.github.com
Reply-To: blackberry/Ripple-UI reply@reply.github.com
Cc: Mark Dineenmddineen@gmail.com
Subject: Re: [Ripple-UI] Emulate Media Queries (#509)

Are you at the hub monday?

I wouldn't mind pairing up to clean some stuff up
Sent on the TELUS Mobility network with BlackBerry

---

Reply to this email directly or view it on GitHub:
https://github.com/blackberry/Ripple-UI/pull/509#issuecomment-8379145

Commit:
removed debug

gtanner committed:
Cleaned up code based on code review

Pair Programmed with Mark

- Got rid of constant (in favour of a closured collection of transforms)
- Put init call in same place as other simular modules
- some cleaning up of code for clarity

gtanner commented:
+1
